### ISSUE TITLE
docker exec is confused with shell's exec

### ISSUE THREAD

**Issue by smedelyan** [koalaman/shellcheck#2490]
#### For bugs
- Rule Id (if any, e.g. SC1000): SC2093, SC2154
- My shellcheck version (`shellcheck --version` or 'online'): online
- [x] I tried on shellcheck.net and verified that this is still a problem on the latest commit
- [ ] It's not reproducible on shellcheck.net, but I think that's because it's an OS, configuration or encoding issue

#### Here's a snippet or screenshot that shows the problem:

```sh
#!/bin/bash

#just an alias to use later
ddb=docker exec -i "${CONTAINER_NAME}" psql -d "${DB_NAME}" -U "${USER}"

$ddb < some.sql
```

#### Here's what shellcheck currently says:
```
Line 4	SC2093: Remove "exec " if script should continue after this command.
Line 6	SC2154: ddb is referenced but not assigned.
```
#### Here's what I wanted or expected to see:

Clean output. `docker exec` is not `exec`, everything is valid.

**Comment by smedelyan** [comment]
Sorry, this is an incorrect script. I should've wrote it like `ddb="docker exec ... "`. Closing the issue
